wagtailvideos/templates/wagtailvideos/videos/index.html

```diff
@@ -23,7 +23,8 @@
     {% trans "Videos" as video_str %}
 
     {% trans "Add a video" as add_video_str %}
-    {% include "wagtailadmin/shared/header.html" with title=video_str add_link="wagtailvideos:add_multiple" icon="media" add_text=add_video_str search_url="wagtailvideos:index" %}
+    {% url "wagtailvideos:add_multiple" as add_link %}
```

Comment:
This should be the name of the route, not a reversed url, it's causing the tests to fail: https://travis-ci.org/github/neon-jungle/wagtailvideos/builds/723598095

You can change the below line to:
```
{% include "wagtailadmin/shared/header.html" with
     title=video_str 
     action_url="wagtailvideos:add_multiple"
     icon="media"
     action_text=add_video_str 
     search_url="wagtailvideos:index" %}
```

Reply:
Got it, I was using https://github.com/wagtail/wagtail/blob/e097cfaab25bfd0b44a5133f4a43a330a0a16629/wagtail/images/templates/wagtailimages/images/index.html#L29 as base, so I thought that it needed to be the reversed url

Reply:
oh yeah you're right, not sure what I was thinking there. Looking at the [template code](https://github.com/wagtail/wagtail/blob/v2.10.1/wagtail/admin/templates/wagtailadmin/shared/header.html) it definitely should be a fully reversed url, it must be something else tripping up the tests

Reply:
Oh I see what it is now, in 2.10 it takes a reversed url but before that it was a just a string you can see 2.9 here: 
https://github.com/wagtail/wagtail/blob/v2.9.3/wagtail/admin/templates/wagtailadmin/shared/header.html

Reply:
Fixed :D